editor/groups_editor.cpp

```diff
@@ -274,20 +274,21 @@ void GroupsEditor::_update_groups_and_tree() {
 	_update_tree();
 }
 
-void GroupsEditor::_update_scene_groups(Node *p_node) {
-	if (scene_groups_cache.has(p_node)) {
-		scene_groups = scene_groups_cache[p_node];
-		scene_groups_cache.erase(p_node);
+void GroupsEditor::_update_scene_groups(ObjectID p_id) {
+	if (scene_groups_cache.has(p_id)) {
+		scene_groups = scene_groups_cache[p_id];
+		scene_groups_cache.erase(p_id);
```

Comment:
Unrelated, but now I noticed this function is inefficient. This would be better:
```suggestion
	HashMap<Node *, HashMap<StringName, bool>>::Iterator I = scene_groups_cache.find(p_node);
	if (I) {
		scene_groups = I->value;
		scene_groups_cache.remove(I);
```